lib/transformers/govukCheckboxes.js

```diff
@@ -18,6 +18,15 @@ module.exports = ({schemaKey, schema, options, transformations, data, schemaErro
         }
     };
 
+    // schema with single checkboxes are assumed to be declaration-style pages
+    // with a large chunk of HTML instead of a small amount of "hint" text.
+    if (schema.items.anyOf.length === 1) {
+        const singleItem = schema.items.anyOf[0];
+        opts.macroOptions.hint = {html: singleItem.description};
+        // stop it from being used in the .map() below.
+        delete singleItem.description;
+    }
```

Comment:
Not a fan of deleting data unless there's no other way around it. It probably wont cause a problem but its a bit more risky that simply including `&& items.anyOf.length > 1` in the logic below to avoid it being used again?